.travis.yml

```diff
@@ -2,7 +2,7 @@ dist: trusty
 language: node_js
 cache: yarn
 node_js:
-- 8
+- 14
```

Comment:
Actually I think we can delete the Travis file, since we'll now be using Github Actions.